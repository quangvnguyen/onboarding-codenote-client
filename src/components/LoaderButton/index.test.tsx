import React from 'react';
import { shallow } from 'enzyme';
import LoaderButton from './index';

test('renders without loading', () => {
  const wrapper = shallow(<LoaderButton />);
  expect(wrapper).toMatchSnapshot();
});

test('renders with loading', () => {
  const wrapper = shallow(<LoaderButton isLoading={true} />);
  expect(wrapper).toMatchSnapshot();
});

test('renders without disabled', () => {
  const wrapper = shallow(<LoaderButton disabled={false} />);
  expect(wrapper).toMatchSnapshot();
});

test('renders with disabled', () => {
  const wrapper = shallow(<LoaderButton disabled={true} />);
  expect(wrapper).toMatchSnapshot();
});
