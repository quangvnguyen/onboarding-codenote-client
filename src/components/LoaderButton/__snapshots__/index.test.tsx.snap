// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with disabled 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  className="LoaderButton "
  disabled={true}
/>
`;

exports[`renders with loading 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  className="LoaderButton "
  disabled={true}
>
  <Glyphicon
    bsClass="glyphicon"
    className="spinning"
    glyph="refresh"
  />
</Button>
`;

exports[`renders without disabled 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  className="LoaderButton "
  disabled={false}
/>
`;

exports[`renders without loading 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  className="LoaderButton "
  disabled={false}
/>
`;
